spec/models/vote_spec.rb

```diff
@@ -1,8 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe Vote, type: :model do
+  let!(:new_article) { create :article, title: 'Comment tester' }
+  let!(:upvote) { create :vote, score: 1, voteable: new_article }
+  let!(:downvote) { create :vote, score: -1, voteable: new_article }
+
   it { should validate_presence_of :voteable }
   it { should belong_to :voteable }
   it { should validate_presence_of :user }
   it { should belong_to :user }
+
+  describe "#upvote?" do
+    it 'checks if vote is upvote' do
+      expect(upvote.upvote?).to eq(true)
+    end
+  end
```

Comment:
A bit pedantic, but this is missing the check for `false` when it's a downvote (and similar for the `#downvote?` test). Might consider something like this:

```ruby
describe "#upvote?" do
  context "is an upvote" do
    subject { upvote }

    it "returns true" do
      expect(subject.upvote?).to eq(true)
    end
  end

  context "is a downvote" do
    subject { downvote }

    it "returns false" do
      expect(subject.upvote?).to eq(false)
    end
  end
end
```